ONSdigital/dp-frontend-renderer
# Job ID is allocated at start of journey to support data persistence

### What

- Add some very basic skeleton templates for cmd user journey
- Define page model and add handlers

### How to review

See https://github.com/ONSdigital/dp-frontend-filter-dataset-controller/pull/1
- Clone this repository and dp-frontend-filter-dataset-controller (checkout branch feature/job-id)
- Ensure sixteens is running
- Run `make debug` within both of the repositories
- Navigate your browser to http://localhost:20001/datasets/1234/editions/5678/versions/2017
- Follow the user journey by "choosing a filter" and clicking the start button
- This will generate you a job id and redirect you to /jobs/{job_id}
- Finish the journey by clicking on preview and download
- Please note that these are only skeleton templates and do not represent the final user journey
- You should also run `make test` to ensure all tests pass successfully

### Who can review

Anyone but preferably a front end dev
